multicluster/config/overlays/leader-ns/labelidentityimport_rbac.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Thanks for the suggestion. The ClusterRole was intended to 1. enable member access 2. enable leader to create such resource. After your suggestion I realized I could add corresponding permissions to `leader-ns/member_cluster_role.yaml` and `leader-ns/role/yaml` respectively. Refactored.

Reply:
After more testing, I have to revert back to the original pattern. The reason for that is 1. Unlike `ResourceImport`, the `LabelIdentityImport` resource is a cluster-scope resource, whose rbac must be specified with a ClusterRole instead of a Role 2. `leader-ns/member_cluster_role.yaml`, with a deceiving name, is actually a Role instead of ClusterRole 3. Adding rbac to `leader-ns/role.yaml` does not work as well, since each leader controller uses a Namespaced SA for creating `LabelIdentityImport` resources. See the errors below:
```
failed to create LabelIdentityImport for new label" err="labelidentityimports.multicluster.crd.antrea.io is forbidden: User \"system:serviceaccount:antrea-multicluster:antrea-mc-controller\" cannot create resource \"labelidentityimports\" in API group \"multicluster.crd.antrea.io\" at the cluster scope"
```
```
failed to list *v1alpha1.LabelIdentityImport: labelidentityimports.multicluster.crd.antrea.io is forbidden: User "system:serviceaccount:antrea-multicluster:antrea-mc-member-access-sa" cannot list resource "labelidentityimports" in API group "multicluster.crd.antrea.io" at the cluster scope
```